src/julia-parser.scm

```diff
@@ -865,7 +865,7 @@
            (begin
              #;(if (and (number? ex) (= ex 0))
                  (error "juxtaposition with literal \"0\""))
-             `(call * ,ex ,(parse-unary s))))
+             `(call .* ,ex ,(parse-unary s))))
```

Comment:
Sure, it's not a big deal.

But why is it a performance disaster for the scalar case?  Shouldn't it be getting inlined to be equivalent to `2 * 2`?